x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/indexlifecycle/action/GetLifecycleAction.java

```diff
@@ -37,42 +38,49 @@ public Response newResponse() {
 
     public static class Response extends ActionResponse implements ToXContentObject {
 
-        private List<LifecyclePolicy> policies;
+        private Map<LifecyclePolicy, Tuple<Long, String>> policiesToMetadata;
```

Comment:
I think we should make this nicer here. Having a map is not nice because the key is not something you are realistically going to to use to lookup things and Tuples are generally not particularly nice to consume. Can we create a wrapper class around the policy, date and version so we have an object that is easier to consume both for us internally and for users of the transport API (given this class will be exposed to the Transport API)?